recipes/vispr/meta.yaml

```diff
@@ -9,25 +9,25 @@ source:
   sha256: dc2cf6ac9c8930b0f1f1b3a2b7f57bfae180b2ac3674060123fd2d7fe10ccd82
 
 build:
-  skip: True # [py27]
+  noarch: python
```

Comment:
noarch python should only be used by pure python packages, numpy is not noarch and not pure python my understanding is that this is not working. Maybe @mbargull knows more?

Reply:
A `noarch` package can have non-`noarch` dependencies, no problem. A good example is the dependency on `python` itself, which is not `noarch` :wink:.
The package itself seems to be pure Python, so this looks good.

Reply:
For Python projects I usually just take a look at the `setup.py` and `setup.cfg` (+ `requirement.txt` if that one is used in `setup.py` and the license file of course).
If `setup.py` has no platform-specific code (including in requirement specifiers) and doesn't build any extensions, the package should usually be pure Python and thus be likely eligible for `noarch: python`.
(Of course there can be cases where `noarch: python` isn't really true, e.g., when \[only for a subset of all covered systems\] some system libraries are dynamically loaded or system-specific file system accesses happen -- but that would require knowing/reading the whole source code ;P)

Reply:
And if the package's `setup.py` is nice (like it's the case for `vispr`) and only imports system modules but not itself or thirdparty packages, we can shrink the `build` and `requirements/host` section to
```yaml
build:
  script: python -m pip install --no-deps --ignore-installed .

requirements:
  host:
   - python
   - pip